### PR TITLE
Updated NOTICE-binary and license files for aea8fca7e2535ce2e8a445754…

### PR DESCRIPTION
**Pull request by mcvsubbu**
…cf5c25bef0cb542

These have been constructed as per instructions in
https://cwiki.apache.org/confluence/display/PINOT/Creating+an+Apache+Release

### CODE REVIEW

File: LICENSE-binary [hunk 1]
```diff
@@ -228,70 +219,41 @@ commons-configuration:commons-configuration:1.6
...
 me.lemire.integercompression:JavaFastPFOR:0.0.13
-net.jpountz.lz4:lz4:1.2.0
 org.apache.avro:avro:1.7.6
 org.apache.commons:commons-compress:1.9
 org.apache.commons:commons-csv:1.0
 org.apache.commons:commons-lang3:3.5
 org.apache.commons:commons-math:2.1
-org.apache.commons:commons-math3:3.2
-org.apache.commons:commons-math3:3.2
-org.apache.curator:curator-client:2.7.1
```
Comment (snleee) [inline LICENSE-binary]: In NOTICE file, you added 
```
Curator Framework
Copyright 2011-2015 The Apache Software Foundation
```
and you remove `org.apache.curator` here.

File: LICENSE-binary [hunk 2]
```diff
@@ -228,70 +219,41 @@ commons-configuration:commons-configuration:1.6
...
 org.apache.httpcomponents:httpclient:4.5.3
 org.apache.httpcomponents:httpcore:4.4.6
 org.apache.httpcomponents:httpmime:4.5.3
-org.apache.kafka:kafka_2.10:0.9.0.1
-org.apache.kafka:kafka-clients:0.9.0.1
 org.apache.logging.log4j:log4j-1.2-api:2.11.2
 org.apache.logging.log4j:log4j-api:2.11.2
 org.apache.logging.log4j:log4j-core:2.11.2
 org.apache.logging.log4j:log4j-slf4j-impl:2.11.2
-org.apache.orc:orc-core:1.5.2
```
Comment (snleee) [inline LICENSE-binary]: Why we are removing orc?

File: LICENSE-binary [hunk 3]
```diff
@@ -228,70 +219,41 @@ commons-configuration:commons-configuration:1.6
...
-org.apache.directory.api:api-asn1-api:1.0.0-M20
-org.apache.directory.api:api-util:1.0.0-M20
-org.apache.directory.server:apacheds-i18n:2.0.0-M15
-org.apache.directory.server:apacheds-kerberos-codec:2.0.0-M15
-org.apache.hadoop:hadoop-annotations:2.2.0
-org.apache.hadoop:hadoop-auth:2.7.0
-org.apache.hadoop:hadoop-common:2.7.0
-org.apache.hadoop:hadoop-hdfs:2.2.0
-org.apache.hadoop:hadoop-mapreduce-client-core:2.2.0
-org.apache.helix:helix-core:0.8.2
```
Comment (snleee) [inline LICENSE-binary]: why we are deleting helix? We definitely uses helix..

Comment (mcvsubbu) [reply LICENSE-binary]: hmm.. something is wrong with the way the automated scripts generate this.... Let me look over again.

**Comment by snleee** [comment]
I checked jar files that we are packaging on the current master's branch(`115cab9649378dcbd6afffafcfa5b0bbca8c24f7`).

It currently pulls 171 libraries and they include most of the libraries that you removed.

```
total 304568
-rw-r--r--  1 snlee  LINKEDIN\eng   153K Jun 18 22:10 JavaFastPFOR-0.0.13.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   322K Jun 18 22:10 RoaringBitmap-0.8.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   131K Jun 18 22:12 aircompressor-0.10.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   301K Jun 18 22:10 antlr4-runtime-4.6.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   4.4K Jun 18 22:12 aopalliance-1.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    14K Jun 18 22:09 aopalliance-repackaged-2.5.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    44K Jun 18 22:11 apacheds-i18n-2.0.0-M15.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   675K Jun 18 22:11 apacheds-kerberos-codec-2.0.0-M15.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    16K Jun 18 22:11 api-asn1-api-1.0.0-M20.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    78K Jun 18 22:11 api-util-1.0.0-M20.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   150K Jun 18 22:10 args4j-2.32.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    42K Jun 18 22:11 asm-3.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   115K Jun 18 22:09 aspectjrt-1.8.4.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   540K Jun 18 22:09 async-http-client-1.9.21.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    20K Jun 18 22:12 audience-annotations-0.5.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   1.2M Jul 15 14:35 avatica-core-1.13.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   416K Jun 18 22:12 avro-1.7.6.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   260K Jul 15 14:37 avro-ipc-1.7.4-tests.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   183K Jul 15 14:37 avro-ipc-1.7.4.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   163K Jul 15 14:37 avro-mapred-1.7.4-hadoop2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    77K Jun 18 22:12 azure-data-lake-store-sdk-2.1.5.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   4.6M Jul 15 14:35 calcite-core-1.19.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   449K Jul 15 14:35 calcite-linq4j-1.19.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   227K Jun 18 22:09 commons-beanutils-1.8.3.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   201K Jun 18 22:09 commons-beanutils-core-1.8.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    40K Jun 18 22:08 commons-cli-1.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   227K Jun 18 22:07 commons-codec-1.6.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   562K Jun 18 22:08 commons-collections-3.2.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   369K Jun 18 22:12 commons-compress-1.9.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   292K Jun 18 22:11 commons-configuration-1.6.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    34K Jun 18 22:10 commons-csv-1.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   140K Jun 18 22:09 commons-digester-1.8.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   298K Jun 18 22:08 commons-httpclient-3.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   181K Jun 18 22:12 commons-io-2.4.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   278K Jun 18 22:07 commons-lang-2.6.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   469K Jun 18 22:07 commons-lang3-3.5.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    60K Jun 18 22:12 commons-logging-1.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   813K Jun 18 22:10 commons-math-2.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   1.6M Jun 18 22:12 commons-math3-3.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   267K Jun 18 22:11 commons-net-3.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   279K Jun 18 22:11 config-1.3.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    68K Jun 18 22:11 curator-client-2.7.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   182K Jun 18 22:11 curator-framework-2.7.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    78K Jun 18 22:10 disruptor-3.3.4.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    18M Jul 15 14:36 fastutil-8.2.3.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   832K Jun 18 22:09 grizzly-framework-2.4.4.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   341K Jun 18 22:09 grizzly-http-2.4.4.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   305K Jun 18 22:09 grizzly-http-server-2.4.4.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   186K Jun 18 22:11 gson-2.2.4.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   2.3M Jun 18 22:12 guava-20.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   694K Jun 18 22:12 guice-3.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    63K Jun 18 22:12 guice-servlet-3.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    16K Jun 18 22:12 hadoop-annotations-2.2.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    69K Jun 18 22:11 hadoop-auth-2.7.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   3.3M Jun 18 22:12 hadoop-common-2.7.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   5.0M Jun 18 22:13 hadoop-hdfs-2.2.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   1.4M Jun 18 22:13 hadoop-mapreduce-client-core-2.2.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   1.5M Aug 12 12:57 helix-core-0.8.4.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   226K Jun 18 22:12 hive-storage-api-2.6.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   183K Jun 18 22:09 hk2-api-2.5.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   184K Jun 18 22:09 hk2-locator-2.5.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   115K Jun 18 22:09 hk2-utils-2.5.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   1.4M Jun 18 22:12 htrace-core-3.1.0-incubating.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   730K Jun 18 22:11 httpclient-4.5.3.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   316K Jun 18 22:11 httpcore-4.4.6.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    40K Jun 18 22:10 httpmime-4.5.3.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    65K Jun 18 22:11 jackson-annotations-2.9.8.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   318K Jun 18 22:10 jackson-core-2.9.8.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   227K Jun 18 22:12 jackson-core-asl-1.9.13.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   1.3M Jun 18 22:11 jackson-databind-2.9.8.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    40K Jun 18 22:10 jackson-dataformat-yaml-2.9.8.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    32K Jun 18 22:10 jackson-jaxrs-base-2.9.8.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    15K Jun 18 22:10 jackson-jaxrs-json-provider-2.9.8.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   762K Jun 18 22:12 jackson-mapper-asl-1.9.13.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    32K Jun 18 22:10 jackson-module-jaxb-annotations-2.9.8.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    25K Jun 18 22:09 jakarta.annotation-api-1.3.4.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   5.3K Jun 18 22:09 jakarta.inject-2.5.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   137K Jun 18 22:09 jakarta.ws.rs-api-2.1.5.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   732K Jun 18 22:10 javassist-3.19.0-GA.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   2.4K Jun 18 22:12 javax.inject-1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    83K Jun 18 22:11 javax.servlet-api-3.0.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   113K Jun 18 22:10 javax.ws.rs-api-2.0.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    80K Jun 18 22:09 jcabi-aspects-0.22.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    49K Jun 18 22:10 jcabi-log-0.17.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   192K Jun 18 22:09 jersey-client-2.28.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   1.1M Jun 18 22:10 jersey-common-2.28.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    40K Jun 18 22:10 jersey-container-grizzly2-http-2.28.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    65K Jun 18 22:10 jersey-container-servlet-core-2.25.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   448K Jun 18 22:12 jersey-core-1.9.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    82K Jun 18 22:09 jersey-entity-filtering-2.28.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    75K Jun 18 22:10 jersey-hk2-2.28.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    84K Jun 18 22:09 jersey-media-jaxb-2.28.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    73K Jun 18 22:10 jersey-media-json-jackson-2.28.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    78K Jun 18 22:10 jersey-media-multipart-2.28.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   696K Jun 18 22:12 jersey-server-1.9.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   914K Jun 18 22:11 jersey-server-2.28.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   527K Jun 18 22:12 jetty-6.1.26.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   173K Jun 18 22:12 jetty-util-6.1.26.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    85K Jun 18 22:12 jline-0.9.94.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   556K Jun 18 22:11 joda-time-2.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    61K Jun 18 22:11 jopt-simple-4.6.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   181K Jun 18 22:11 jsch-0.1.42.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    32K Jun 18 22:11 jsr305-3.0.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    45K Jun 18 22:10 jsr311-api-1.1.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   118K Jun 18 22:08 junit-3.8.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   626K Jun 18 22:11 kafka-clients-0.9.0.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   4.7M Jun 18 22:11 kafka_2.10-0.9.0.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   350K Jun 18 22:12 kryo-shaded-3.0.3.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   318K Jun 18 22:10 larray-0.2.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    14K Jun 18 22:10 larray-buffer-0.2.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    32K Jun 18 22:10 larray-mmap-0.2.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   241K Jul 29 15:12 libthrift-0.12.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    63K Jun 18 22:10 log4j-1.2-api-2.11.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   260K Jun 18 22:10 log4j-api-2.11.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   1.6M Jun 18 22:10 log4j-core-2.11.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    23K Jun 18 22:10 log4j-slf4j-impl-2.11.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   162K Jun 18 22:11 lz4-1.2.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    80K Jun 18 22:11 metrics-core-2.2.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   133K Jun 18 22:09 metrics-core-3.2.3.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    66K Jun 18 22:10 mimepull-1.9.11.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   5.6K Jun 18 22:12 minlog-1.3.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   1.3M Jun 18 22:12 netty-3.9.6.Final.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   3.7M Jun 18 22:12 netty-all-4.1.28.Final.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    41K Jun 18 22:12 objenesis-2.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   792K Jun 18 22:12 orc-core-1.5.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    48K Jun 18 22:12 orc-mapreduce-1.5.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    27K Jun 18 22:12 orc-shims-1.5.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    20K Jun 18 22:09 osgi-resource-locator-1.0.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    29K Jun 18 22:11 paranamer-2.3.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    37K Aug 12 12:58 pinot-api-0.2.0-SNAPSHOT.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    15K Aug 12 13:00 pinot-azure-filesystem-0.2.0-SNAPSHOT.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   167K Aug 12 12:58 pinot-broker-0.2.0-SNAPSHOT.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    31M Aug 12 12:57 pinot-common-0.2.0-SNAPSHOT-shaded.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   908K Aug 12 12:57 pinot-common-0.2.0-SNAPSHOT.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    54K Aug 12 12:58 pinot-connector-kafka-0.9-0.2.0-SNAPSHOT.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    23K Aug 12 12:58 pinot-connector-kafka-base-0.2.0-SNAPSHOT.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   1.1M Aug 12 12:58 pinot-controller-0.2.0-SNAPSHOT.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    18M Aug 12 12:58 pinot-core-0.2.0-SNAPSHOT-tests.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   1.5M Aug 12 12:57 pinot-core-0.2.0-SNAPSHOT.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    81K Aug 12 12:58 pinot-hadoop-0.2.0-SNAPSHOT.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    16K Aug 12 13:00 pinot-hadoop-filesystem-0.2.0-SNAPSHOT.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    16K Aug 12 13:00 pinot-orc-0.2.0-SNAPSHOT.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    79K Aug 12 12:58 pinot-server-0.2.0-SNAPSHOT.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   3.0M Aug 12 12:58 pinot-tools-0.2.0-SNAPSHOT.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   133K Aug 12 12:57 pinot-transport-0.2.0-SNAPSHOT.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   521K Jun 18 22:12 protobuf-java-2.5.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   128K Jun 18 22:10 reflections-0.9.11.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   6.8M Jun 18 22:11 scala-library-2.10.5.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   131K Jun 18 22:10 servlet-api-2.5-20081211.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   3.9K Jun 18 22:10 shims-0.8.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    40K Jun 18 22:12 slf4j-api-1.7.25.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   263K Jun 18 22:11 snakeyaml-1.16.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   580K Jun 18 22:11 snappy-java-1.1.1.7.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   170K Jun 18 22:10 stream-2.7.0.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    20K Jun 18 22:10 swagger-annotations-1.5.16.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   109K Jun 18 22:10 swagger-core-1.5.16.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    69K Jun 18 22:10 swagger-jaxrs-1.5.16.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   6.6K Jun 18 22:10 swagger-jersey2-jaxrs-1.5.16.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   140K Jun 18 22:10 swagger-models-1.5.16.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   1.5M Jun 18 22:11 swagger-ui-2.2.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    50K Jun 18 22:10 t-digest-3.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   1.8M Jun 18 22:10 tyrus-standalone-client-1.5.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    91K Jun 18 22:10 validation-api-2.0.1.Final.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   800K Jun 18 22:11 vavr-0.9.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    26K Jun 18 22:10 vavr-match-0.9.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   439K Jun 18 22:08 velocity-1.7.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   363K Jun 18 22:10 xerial-core-2.1.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   561K Jun 18 22:10 xerial-core-3.2.2.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    15K Jun 18 22:12 xmlenc-0.52.jar
-rw-r--r--  1 snlee  LINKEDIN\eng    72K Jun 18 22:11 zkclient-0.7.jar
-rw-r--r--  1 snlee  LINKEDIN\eng   867K Jun 18 22:12 zookeeper-3.4.11.jar
```

**Comment by mcvsubbu** [comment]
Updated the instructions, will start over.

**Comment by snleee** [comment]
@mcvsubbu When is the commit `aea8fca7e2535ce2e8a445754`?

**Pull request closed without merging**